### DISCUSSION TITLE
multi calc outputs (reuse file IO)

### DISCUSSION THREAD

**Discussion by tooptoop4** [dathere/qsv#2149] [Q&A] [answered]
how to get 2 computations done on 1 disk read?
ie instead of sum_result=$(cat file | sum column1); crc32_result=$(cat file | crc32 column1);   can both the sum and crc32 of the column be done off single disk read?

**Comment by jqnatividad** [comment, marked as answer]
Hi @tooptoop4 ,
Yes you can, with the `luau` command.

https://github.com/jqnatividad/qsv/blob/97b56c9b0ea8171987c163599e208496810d782c/src/cmd/luau.rs#L50-L51

And you can even have a full blown luau script:

https://github.com/jqnatividad/qsv/blob/97b56c9b0ea8171987c163599e208496810d782c/tests/test_luau.rs#L1608-L1696
